packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/reactnative/signout.md

```diff
@@ -0,0 +1,3 @@
+ReactNative signout
```

Comment:
@nbarbettini - With guides merged now, should we remove all react-native content until it is ready to display?  Or should we have metadata to say what languages to show instead of basing it off of the content?  ( *grimace at all the wordy metadata* )

Reply:
I don't have permissions to update this PR. Can we merge this and create another PR to remove react native from master?

Reply:
Merging it would publish to live.  You're better off putting the non-react-native content into a new PR.